components/Main/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import { View, Text, TextInput, TouchableOpacity, ScrollView } from 'react-native'
+import Item from 'components/Item'
+import './index.styl'
+import { observer } from 'startupjs'
+
+export default observer(function App () {
+
+  const [input, setInput] = useState('')
+  const[item, setItem] = useState([])
```

Comment:
@TimmyByDeOverly why do you need an item variable and even an array? is it a list? then items. But you must store todos in the database.

Reply:
done